.github/workflows/stress-test.yml

```diff
@@ -0,0 +1,35 @@
+# Nightly Stress Test for self-hosted runners
+name: Self-hosted Runners Nightly Stress Test
```

Comment:
Instead of "stress" maybe we should call this a scale test.  We don't actually do anything with the runners which makes me lean a little away from calling it stress long term.

Reply:
Changed to scale.